src/states/elder/mod.rs

```diff
@@ -1697,14 +1697,15 @@ impl Approved for Elder {
             );
             return Ok(());
         }
-        if self.chain.is_churn_in_progress() {
-            debug!(
-                "{} - Trying to carry out parsec pruning during churning.",
+        if self.chain.membership_change_in_progress() {
```

Comment:
I don't think this is correct, the reason we can ignore prune when elder member change is a new parsec is created at the end. not the case for relocation.

So probably what need to happen is relocation either work over pruned parsec, or prune still work.

Reply:
Considering the tests are passing, would it be worth raising an issue for this and merge this PR before you are on holiday? @madadam

Reply:
OK, will do. But I might still be able to raise it as a separate PR before my leave.

Reply:
https://github.com/maidsafe/routing/issues/1963